docs/Operators.md

```diff
@@ -15137,20 +15137,67 @@ This version of the operator has been available since version 1 of the default O
 #### Examples
 
 <details>
-<summary>unsqueeze</summary>
+<summary>unsqueeze_one_axis</summary>
 
 ```python
+x = np.random.randn(3, 4, 5).astype(np.float32)
+
+for i in range(len(x) + 1):
+    node = onnx.helper.make_node(
+        'Unsqueeze',
+        inputs=['x'],
+        outputs=['y'],
+        axes=[i],
+    )
+    y = np.expand_dims(x, axis=i)
+
+    expect(node, inputs=[x], outputs=[y],
+           name='test_unsqueeze_axis_' + str(i))
+```
+
+</details>
+
+
+<details>
+<summary>unsqueeze_three_axes</summary>
+
+```python
+x = np.random.randn(3, 4, 5).astype(np.float32)
+
 node = onnx.helper.make_node(
     'Unsqueeze',
     inputs=['x'],
     outputs=['y'],
-    axes=[0],
+    axes=[2, 4, 5],
 )
+y = np.expand_dims(x, axis=2)
+y = np.expand_dims(y, axis=4)
+y = np.expand_dims(y, axis=5)
```

Comment:
I am not sure if this is the valid test. The original tensor is 3-D, so the min axis you can expand is 0 and the max one is 3. Did I miss something?